quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliCreateJvmApplicationIT.java

```diff
@@ -175,4 +188,26 @@ private void assertInstalledExtensions(QuarkusCliRestService app, String... expe
                 expectedExtension + " not found in " + extensions));
     }
 
+    private void assertExpectedJavaVersion(File pomFile) {
+        MavenXpp3Reader reader = new MavenXpp3Reader();
+        String javaVersion = getSystemJavaVersion();
+        try {
+            Model model = reader.read(new FileReader(pomFile));
+            Assertions.assertEquals(model.getProperties().get("maven.compiler.release"), javaVersion,
```

Comment:
I think this is true only for Java LTS versions - 11 and 17.

If I use Java 15, generated project will be set to 11. Same if I use Java 18, generated project will be set to 17. Closes lower or equal LTS version gets used. We are using 11 and 17, so fine for us.

I don't think it's necessary to adjust the check.